rosidl_typesupport_opensplice_c/resource/srv__type_support_c.cpp.em

```diff
@@ -30,6 +30,7 @@
 #include "@(spec.pkg_name)/srv/dds_opensplice/ccpp_Sample_@(spec.srv_name)_Response_.h"
 #include "rosidl_typesupport_opensplice_c/identifier.h"
 
+// TODO(dirk-thomas) including another cpp file is just nasty
```

Comment:
Yes, because the following include of a cpp file is really bad practice and should be fixed.